spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/RateLimitKeyer.java

```diff
@@ -0,0 +1,11 @@
+package com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.config;
+
+import com.marcosbarbero.cloud.autoconfigure.zuul.ratelimit.config.properties.RateLimitProperties;
+import org.springframework.cloud.netflix.zuul.filters.Route;
+
+import javax.servlet.http.HttpServletRequest;
+
+public interface RateLimitKeyer {
```

Comment:
I think a name like `KeyGenerator` or `RateLimitKeyGenerator` will fit better.
WDYT?

Reply:
I went through several combinations of method/class/package names to use and wasn't particularly happy with any of them.

`RateLimit` seems to be a prefix used so will update to `RateLimitKeyGenerator`.